explore/src/main/scala/explore/highcharts.scala

```diff
@@ -28,6 +27,6 @@ def commonOptions: ChartOptions =
   ChartOptions()
     .setStyledMode(true)
     .setAlignTicks(false)
-    .setZooming(ChartZoomingOptions().setType(OptionsTypeValue.xy))
+    .setZooming(ChartZoomingOptions().setType(highchartsStrings.xy.toString))
     .setPanning(ChartPanningOptions().setEnabled(true))
-    .setPanKey(OptionsPanKeyValue.shift)
+    .setPanKey("shift")
```

Comment:
There's no constant anymore for this.

Reply:
WDYM, on the facade? Maybe we should have our own

Reply:
Yup, ST is not generating it anymore.

We can add our own to the facade, yes.